runtime/binding-tcp/src/test/java/io/aklivity/zilla/runtime/binding/tcp/internal/bench/TcpServerBM.java

```diff
@@ -61,19 +66,34 @@
     //private long routedId;
 
     {
-        Properties properties = new Properties();
-        properties.setProperty(ENGINE_DIRECTORY.name(), "target/zilla-benchmarks");
-        properties.setProperty(ENGINE_STREAMS_BUFFER_CAPACITY.name(), Long.toString(1024L * 1024L * 16L));
+        Properties props = new Properties();
+        props.setProperty(ENGINE_DIRECTORY.name(), "target/zilla-benchmarks");
+        props.setProperty(ENGINE_STREAMS_BUFFER_CAPACITY.name(), Long.toString(1024L * 1024L * 16L));
 
-        final Configuration configuration = new Configuration(properties);
+        final EngineConfiguration config = new EngineConfiguration(props);
+        URL configURL = config.configURL();
+        if ("file".equals(configURL.getProtocol()))
+        {
+            Path configPath = Paths.get(configURL.getPath());
+            if (configPath.endsWith("zilla.yaml") && Files.notExists(configPath))
+            {
+                String configJSON = String.format("file:%s", configPath.resolveSibling("zilla.json"));
+                props.setProperty(ENGINE_CONFIG_URL.name(), configJSON);
+                configPath = Paths.get(config.configURL().getPath());
+                System.out.println("zilla.yaml file not found, loading zilla.json instead");
+            }
+            if (configPath.getFileName().toString().endsWith(".json"))
+            {
+                System.out.println("warning: json syntax is deprecated, migrate to yaml");
+            }
+        }
```

Comment:
We don't need to replicate this backwards compatibility code from `zilla start` in the benchmark, please remove it.

The integration with OpenJMH will add support for picking up a configuration similar to ITs, so it won't be coming from command line or be file based.